packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -87,6 +87,6 @@ export interface FeatureToggles {
   sessionRemoteCache?: boolean;
   disablePrometheusExemplarSampling?: boolean;
   alertingBacktesting?: boolean;
+  editPanelCSVDragAndDrop?: boolean;
   alertingNoNormalState?: boolean;
-  azureMultipleResourcePicker?: boolean;
```

Comment:
Was this not added to the feature toggle registry?

Reply:
Looks like it was.

Reply:
Should be fixed now after merging main.